src/Build.UnitTests/BackEnd/SdkResolution_Tests.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Microsoft.Build.Engine.UnitTests.BackEnd
 {
+    /*
```

Comment:
Can these be adapted? Or delete the file if not.

Reply:
My next commit has renamed and updated these tests